man/veritytab.xml

```diff
@@ -101,6 +149,12 @@ This is based on crypttab(5).
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>hash=hash</option></term>
+
+        <listitem><para>Hash algorithm for dm-verity. For default see veritysetup --help.</para></listitem>
```

Comment:
`<command>…`

mabe say possible values (or at least some of them)

Reply:
done.